### DISCUSSION TITLE
New to this - may be being stupid but i don't get it

### DISCUSSION THREAD

**Discussion by TobyWiddows** [OpenSurv/OpenSurv#17] [General]
Hi,  
I have just  fallen into the world of camera streams,  i have a cheap camera, i can view the feed using rtsp and vlc.  I can get it working in Shinobi.

I am by no means an expert in Linux, I can just about enough to get by.  Saying that i have set up a plex server, mail server, and some homebaked scripts to process files with ffmpeg.

when i found this project I thought this would do what i want it to do.

However, I have installed on an old dell  3020, ubuntu 22.04.05LTS and I don't get what I'm supposed to do now.  Your instructions stop at 
"Run sudo ./install.sh ( this works locally as well as over an ssh session )
Enjoy the demo showcase"  

but nothing happens.

After fiddling and googling, i came across a post, about logging out of Linux and clicking the cog to change the desktop window manager, and Ifound "opensurv" as an option.  But on logging in nothing magical  happens, a part from a message about not being able to find /home/opensurv/bin/opensurv  or similar.

enabled and reset the password on the opensurv user, and when I login to the opensurv desktop with that, it goes black and logs me out again.

Do you have any really basic instructions for first time users to this?

**Comment by SvenVD** [comment]
Hi, welcome!, we first need to make sure you are running  Ubuntu 24.04 LTS and not  Ubuntu 22.04 LTS. 

In the happy path, after sudo ./install.sh it should display the demo showcase and after that you can configure your own streams: https://github.com/opensurv/opensurv?tab=readme-ov-file#configuration. To apply configuration changes you need to restart ([link](https://github.com/OpenSurv/OpenSurv/blob/e904ccc128fe56990242b801af3437302a5d79d1/README.md?plain=1#L102) opensurv.

If there is nothing happening after install.sh then something went wrong, please paste content of /home/opensurv/logs/main.log here so we can trace the root cause.

**Comment by TobyWiddows** [comment]
You are correct i was running 22.04.... my bad i thought i had upgraded to 24.04.

I have been having a  play to day, based on the comments in the PI 3 testing, i tried it on that, i loaded the lite bookworm, set it to autologin, installed python3.yml followed by opensurv.  not a lot happened until I restarted lightdm.service.  then i got a very slow demo, which is to be expected on a pi.  

So i ditched the PI and upgraded my old del ot 24.04 of ubuntu.  set it to autoinstall, installed opensurv, picked lightdm when it gave me a choice between that an gdm3. 

and low and behold i had a demo running.

i have now modified the monitor1.yml (fussy that is!!) and have the main stream and substream from my camera running...

Also have installed shinobi on the same machine connected to the same streams.

perfect..

off to by a beelink now.

**Comment by SvenVD** [comment]
great news!
